record-and-playback/screenshare/scripts/publish/screenshare.rb

```
@@ -37,8 +37,8 @@
 end
 
 # Load parameters and set up paths
-props = YAML::load(File.open(File.expand_path('../../bigbluebutton.yml', __FILE__)))
-screenshare_props = YAML::load(File.open(File.expand_path('../../screenshare.yml', __FILE__)))
+props = YAML::safe_load(File.read(File.expand_path('../../bigbluebutton.yml', __FILE__)))
```

RuboCop (1.86.2)

[convention] Do not use `::` for method calls.

(Style/ColonMethodCall)